en/file_formats/rally_points.md

```diff
@@ -0,0 +1,51 @@
+# Rally (Safe) Point Plan File Format (Mission Protocol)
+
+Rally point plan files are formatted using JSON, and contain information about safe points.
+
+<!--  doc originates from *QGroundControl Dev Guide*: https://dev.qgroundcontrol.com/en/file_formats/rally.html -->
+
+
+```
+{
+    "fileType": "RallyPoints",
+    "groundStation": "QGroundControl",
+    "points": [
+        [
+            47.634309760000001,
+            -122.08936869999999,
+            50
+        ],
+        [
+            47.634244700000004,
```

Comment:
Do points just map to https://mavlink.io/en/messages/common.html#MISSION_ITEM  x, y, z values?